rosbridge_server/launch/rosbridge_websocket.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
   <arg name="port" default="9090" />
-  <arg name="address" default="" />
+  <arg name="address" default="0.0.0.0" />
```

Comment:
A point of self-review: we need to be able to handle an empty "address" param for backwards compatibility.